single-degree.php

```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
+				<a class="media-background-container d-block h-100 py-3 px-3 px-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
```

Comment:
I'm going back and forth on updating this if the 'Did you know' text is updated to an `h5` class, but we could increase the vertical padding for large+ displays here to `py-lg-4` to give it a little extra room without it looking too big